src/components/elements/annotationSidePanel/components/annotationWizard/AnnotationWizard.tsx

```diff
@@ -25,8 +25,18 @@ const AnnotationWizard = (props: Props) => {
 
     /* Base variables */
     const [selectedTabIndex, setSelectedTabIndex] = useState<number>(0);
+
+    /**
+     * Function to go the next step in the wizard
+     */
+    const NextStep = () => {
```

Comment:
I had that function, but already replaced it in the upcoming PR where its now: GoToStrep, which can either mean go forward or backward. Made this and some other little improvements in the next PR, it is a bit mingled because we split up the stories.